### DISCUSSION TITLE
how to start with theia.cloud

### DISCUSSION THREAD

**Discussion by wss29** [eclipse-theia/theia-cloud#302] [Q&A]
hi,
   I follow this link https://theia-cloud.io/documentation/trytheiacloud/#on-your-machine-with-minikube to install my theia.cloud in Windows 10 system
1. download and install miniskube from https://minikube.sigs.k8s.io/docs/start/
2. download and install virtualbox from https://www.virtualbox.org/wiki/Downloads
3. execute 'minikube start --vm=true --driver=virtualbox --download-only" in windows cmd
4. download and install Terraform as described here: https://developer.hashicorp.com/terraform/install, and set system enviroment variable path to Terraform directory
5. check out git clone https://github.com/eclipsesource/theia-cloud.git in vscode
6. cd to terraform/configurations/minikube_getting_started/ and execute terraform init and terraform apply
7. when I execute terraform apply I get error "Plugin did not respond" the error is same as https://github.com/eclipsesource/theia-cloud/discussions/235
8. I read discussions 235, it says that delete lock file, modules folder and retry will ok, 
9. I follow that way, but get error when run "terraform init" as follow image shows,
![image](https://github.com/eclipsesource/theia-cloud/assets/4537809/93df8b31-7391-4b2b-a334-24b3b262b252)
it seems that after I delete modules folder under terraform, "terraform  init" do not download modules again, Does any one knews how to install theia.cloud in windows 10, I have tried many times following documentation steps, but do not success

**Comment by jfaltermeier** [comment]
Can you try again with these cleanup steps please: https://github.com/eclipsesource/theia-cloud/discussions/274#discussioncomment-8207648
Unfortunately I don't know why the minikube/terraform integration seems a bit unstable on Windows
